addons/mail/__manifest__.py

```diff
@@ -118,6 +118,15 @@
             'mail/static/src/model/*.js',
             'mail/static/src/core_models/*.js',
         ],
+        'mail.assets_messaging_frontend': [
+            ('include', 'mail.assets_messaging'),
+            'mail/static/src/models/frontend/*.js',
+        ],
+        'mail.assets_messaging_backend': [
+            ('include', 'mail.assets_messaging'),
+            'mail/static/src/backend_components/*/*',
+            'mail/static/src/models/backend/*.js',
+        ],
```

Comment:
Bundle names are misleading (frontend suppose it's portal/website, which is not the case here), and not that clear.

The `frontend` here is just used for public discuss, so can just dump there (and name folder as `public_discuss`, which is the coarse feature)

for backend, IMO just `web` would be more meaningful, as the context is `/web` on the url